python/data_extraction/export_data.py

```diff
@@ -23,23 +24,82 @@ def __transform_content(input_filename="data/content.json.gz",
             content_generator = json_arrays.read_json(input_file)
             json_arrays.write_json(output_file, transform_function(content_generator))
 
+def get_all_content():
```

Comment:
can this one be private - i.e. prefix  with __

Reply:
I'm still unconvinced about the benefit of this, but fair enough, for consistency at least.